src/standard_libraries/test_re.py

```diff
@@ -23,3 +23,8 @@ def test_re():
     # When only simple capabilities are needed, string methods are preferred because they are
     # easier to read and debug:
     assert 'tea for too'.replace('too', 'two') == 'tea for two'
+    regex = r'\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}\b'
```

Comment:
Could you do the below things here:

1. Can you format the regex something like [verbose formatting](https://diveintopython3.problemsolving.io/regular-expressions.html#verbosere)
2. Can we do something, If we are to introduce case-insensitivity?
3. Can we add one more test for a regex that would match Employee Card numbers? The format of the card number is (XX-XXXXX-44-XX)  -- Here X could be any number.

Reply:
1. i have converted the regex from raw format to verbose format
2. we can remove the 'A-Z' to make it case sensitive. i have updated the regex to only accept lower case now
3. i have added this as well